pkg/controller/plan/kubevirt.go

```diff
@@ -1584,6 +1585,29 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	if err != nil {
 		return
 	}
+	if vm.RootDisk != "" {
+		vmInventory := &model.VM{}
+		err = r.Source.Inventory.Find(vmInventory, vm.Ref)
+		if err != nil {
+			err = liberr.Wrap(err, "vm", vm.Ref.String())
+			return
+		}
+
+		var rootDisk string
+		for diskNum, disk := range vmInventory.Disks {
+			if disk.File == vm.RootDisk {
+				rootDisk = fmt.Sprintf("/dev/sd%c", 'a'+(diskNum)%26)
```

Comment:
consts?
also, i saw germano's comment on the other PR (https://github.com/kubev2v/forklift/pull/872) - he is right regarding the need to have more than 26 disks.
in that case maybe we can put the path generation into a single place and re-use it in both?